packages/core/test-helpers/core-test-helpers-kbn-server/src/create_serverless_root.ts

```diff
@@ -70,7 +76,10 @@ function createServerlessES() {
       await es.runServerless({
         basePath: Path.join(REPO_ROOT, '.es/es_test_serverless'),
       });
+      // runServerless doesn't wait until the nodes are up
+      await waitUntilClusterReady(getServerlessESClient());
```

Comment:
As explained, `es.runServerless` returns as soon as the docker containers as running, which causes tests to potentially fail if they need to access ES as soon as `startES` has been called. 

This naive awaiting solves it.